packages/cli/src/eventbus/event.types.ts

```diff
@@ -310,4 +300,21 @@ export type Event = {
 		user: UserLike; // audit and telemetry
 		publicApi: boolean; // telemetry only
 	};
+
+	'workflow-post-execute': {
+		// audit and telemetry
+		userId?: string;
+
+		// audit only
+		executionId: string;
+		success: boolean;
+		workflowId: string;
+		isManual: boolean;
+		workflowName: string;
+		metadata?: Record<string, string>;
+
+		// telemetry only
+		workflow: IWorkflowBase;
+		runData?: IRun;
```

Comment:
This could be simplified to

```suggestion
		userId?: string;
		executionId: string;
		workflow: IWorkflowBase;
		runData?: IRun;
```

as `metadata` doesn't seem to be passed in any `emit` and all the other fields can be derived from `workflow` and `runData`. That way there's no need to duplicate the logic for deriving those fields